Tests/SwiftDocCTests/LinkTargets/LinkDestinationSummaryTests.swift

```diff
@@ -49,7 +49,7 @@ class ExternalLinkableTests: XCTestCase {
                    }
                    
                    @Section(title: "Create a New AR Project 💻") {
-                      @ContentAndMedia(layout: vertical) {
+                      @ContentAndMedia {
```

Comment:
Can we leave this test unchanged for now? Otherwise this PR looks good to go. Thank you!

```suggestion
                      @ContentAndMedia(layout: vertical) {
```

Reply:
It's ok, I've discard it. But I'm not sure why shouldn't we chsnge it now